packages/@aws-cdk/aws-stepfunctions-tasks/test/invoke-function.test.ts

```diff
@@ -14,19 +14,33 @@ test('Lambda function can be used in a Task', () => {
     handler: 'index.hello',
     runtime: lambda.Runtime.Python27,
   });
-  const task = new sfn.Task(stack, 'Task', { task: new tasks.InvokeFunction(fn) });
+  const task = new sfn.Task(stack, 'Task', {
+    task: new tasks.InvokeFunction(fn, { waitForTaskToken: false })
```

Comment:
We usually try not to change existing tests, but rather add new tests. That's a good way to ensure that we didn't break anything. Since `waitForTaskToken` is false by default, I'd expect this test to remain untouched throughout this change and if it is changed, we need to understand if the change makes sense.